Updated dependencies, now unable to build project due to rss.xml

### Describe the bug

running `pnpm run build` gives

```
20:34:49 [ERROR] [vite] x Build failed in 3.74s
[vite:build-import-analysis] [plugin vite:build-import-analysis] pages/_prefix_/rss.xml.astro.mjs (1:1): Parse error @:1:1

1  |  import { o as objectType, a as stringType, u as unionType, n as numberType, d as dateType, b as arrayType, r as recordType, f as functionType, p as promiseType, c as anyType, e as preprocessType, g as booleanType } from '../../chunks/user-config_CU5WHklU.mjs';
   |   ^
2  |  import { y as yellow } from '../../chunks/astro/server_gqeVL1Yw.mjs';
3  |  import { c as config, g as getBlogEntries, a as context, b as getPathWithBase } from '../../chunks/content_D1pq9u3f.mjs';

  Stack trace:
    at parse$f (file:///Users/visual/Developer/ProjectSources/ChefKiss/ChefKissInc.github.io/node_modules/.pnpm/vite@5.3.4_@types+node@20.14.11/node_modules/vite/dist/node/chunks/dep-D8YhmIY-.js:20510:585)
    at file:///Users/visual/Developer/ProjectSources/ChefKiss/ChefKissInc.github.io/node_modules/.pnpm/rollup@4.19.0/node_modules/rollup/dist/es/shared/node-entry.js:19808:40
 ELIFECYCLE  Command failed with exit code 1.
```

### To Reproduce

1. `pnpm up --latest`
2. `pnpm run build`

### Expected behavior

site builds successfully

### How often does this bug happen?

Every time

### System Info

- OS: macOS
- OS version: 15.0 Beta (24A5289h)
- CPU: M3

### Additional Context

<img width="1624" alt="Screenshot 2024-07-20 at 20 46 30" src="https://github.com/user-attachments/assets/c399c5fb-8758-4fb5-83eb-4cfb66209ac1">

--- a/.github/workflows/integration.yml
+++ b/.github/workflows/integration.yml
@@ -31,19 +31,6 @@ jobs:
       - name: Install dependencies
         run: pnpm install
 
-      - name: Store Playwright Version
-        run: |
-          PLAYWRIGHT_VERSION=$(pnpm playwright --version | sed 's/Version //')
-          echo "PLAYWRIGHT_VERSION=$PLAYWRIGHT_VERSION" >> $GITHUB_ENV
-        working-directory: packages/starlight-blog
-
-      - name: Setup Playwright cache
-        id: cache-playwright
-        uses: actions/cache@v3
-        with:
-          key: playwright-${{ env.PLAYWRIGHT_VERSION }}
-          path: ~/.cache/ms-playwright
-
       - name: Generates TypeScript types
         run: pnpm astro sync
         working-directory: docs

--- a/packages/starlight-blog/libs/rss.ts
+++ b/packages/starlight-blog/libs/rss.ts
@@ -46,7 +46,7 @@ export async function getRSSOptions(site: URL | undefined) {
 }
 
 function getRSSTitle() {
-  let title = (typeof context.title === 'string' ? context.title : context.title[context.defaultLocale]) ?? ''
+  let title = typeof context.title === 'string' ? context.title : context.title[context.defaultLocale] ?? ''
 
   if (title.length > 0) {
     title += ` ${context.titleDelimiter ?? '|'} `

